js/ui/popup/controllers/search.js

```diff
@@ -34,6 +34,12 @@
      */
     angular.module('passmanExtension')
         .controller('SearchCtrl', ['$scope', function ($scope) {
+            API.runtime.sendMessage(API.runtime.id, {method: 'getMasterPasswordSet'}).then(function (MasterPasswordSet) {
+                if (!MasterPasswordSet) {
```

Comment:
wouldn't that check be required for every controller as well as in the right click menu?
maybe you'll find a good position in an api lib for it, to avoid redundancy (if it will be called at multiple controllers)